es78x/src/main/scala/tech/beshu/ror/es/request/context/types/SqlIndicesEsRequestContext.scala

```diff
@@ -47,21 +51,48 @@ class SqlIndicesEsRequestContext private(actionRequest: ActionRequest with Compo
   override protected def indicesFrom(request: ActionRequest with CompositeIndicesRequest): Set[IndexName] =
     sqlIndices.indices.flatMap(IndexName.fromString)
 
+  /** fixme: filter is not applied.
```

Comment:
are you able to write a test which fails for this concrete example? we will @ignore it atm, but will be easier to fix it in future